### ISSUE TITLE
possibly overriding environment variables?

### ISSUE THREAD

**Issue by megantaylor** [matt-oakes/redux-devtools-expo-dev-plugin#6]
so excited and grateful for this plugin, thank you so much!

i installed and set up the plugin following the method described here: https://github.com/reduxjs/redux-devtools/issues/1533#issuecomment-2146390042 and it works!

however, i am now getting typescript errors about all of my environment variables. i poked around, and saw 
```
declare global {
  const process: {
    env: {
      NODE_ENV: "development" | "production";
    };
  };
}
```
in src/index.ts.

looking at some other expo dev plugins (https://github.com/expo/dev-plugins), i notice that they don't do this, and assume the environment variables will be available via metro transform plugins.

so it seems to me as though that global declaration is overriding the env variables i have set up? am i totally off base? is there something i can do on my end to fix this?

**Comment by matt-oakes** [comment]
Hi there. Thanks for reporting this.

So I can look into this a bit easier, can you provide a [minimal reproducable example](https://stackoverflow.com/help/minimal-reproducible-example) of the issue? A Github repo with an Expo app which does nothing but show this issue would be perfect.

**Comment by zakwalters** [comment]
Forgot to check the issues before opening, but #7 has a fix for this.

**Comment by matt-oakes** [comment]
Fixed in [`v0.2.0`](https://github.com/matt-oakes/redux-devtools-expo-dev-plugin/releases/tag/v0.2.0). Thank you for reporting this.
